<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>801</width>
      <height>561</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>801</width>
      <height>561</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_O"/>
    <addaction name="actionSave_S"/>
    <addaction name="action_Save_AS"/>
    <addaction name="separator"/>
    <addaction name="action_E_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menu_Edit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_O">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave_S">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_E_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Save_AS">
   <property name="text">
    <string>&amp;Save as</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
